app/views/list/list.erb

```diff
@@ -143,7 +143,7 @@
 <% if @category %>
   <% content_for :head do %>
     <%= auto_discovery_link_tag(:rss, { action: :category_feed }, rel: 'alternate nofollow', title: t('rss_topics_in_category', category: @category.name)) %>
-    <%= raw crawlable_meta_data(title: @category.name, description: @category.description, image: @category.uploaded_logo&.url.presence) %>
+    <%= raw crawlable_meta_data(title: @category.name, description: @category.description, image: @category.uploaded_logo_dark&.url.presence) %>
```

Comment:
I don't think we need to make this change, the crawlable meta data should use the regular logo.